contracts/multisig/src/workerset.rs

```diff
@@ -0,0 +1,52 @@
+use std::collections::BTreeSet;
+
+use crate::key::PublicKey;
+use crate::msg::Signer;
+use axelar_wasm_std::Participant;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{HexBinary, Uint256};
+use sha3::{Digest, Keccak256};
+
+#[cw_serde]
+pub struct WorkerSet {
+    pub signers: BTreeSet<Signer>,
+    pub threshold: Uint256,
+    // for hash uniqueness. The same exact worker set could be in use at two different times,
+    // and we need to be able to distinguish between the two
```

Comment:
Can you expand on this comment and write a description of why this can cause a problem? Something like "without a nonce, updating to a worker set that is the exact same as a worker set in the past will be immediately confirmed. TODO add nonce to the voting verifier and to the evm gateway"